### PR TITLE
More test logging, and update for Kubeaddons v0.20.3

### PR DESCRIPTION
**Pull request by shaneutt**
**What type of PR is this?**
Chore

**What this PR does/ why we need it**:
Adds more logging for CI tests, and updates to latest `v0.20.3` Kubeaddons which includes fixes for test utils to ensure upgrade tests run properly.

**Checklist**

* [x] The commit message explains the changes and why are needed.
* [x] The code builds and passes lint/style checks locally.
* [x] The relevant subset of integration tests pass locally.
* [x] The core changes are covered by tests.
* [x] The documentation is updated where needed.

### CODE REVIEW

**Review by joejulian — APPROVED** [review]

File: test/addons_test.go [hunk 1]
```diff
@@ -281,14 +286,18 @@ func testgroup(t *testing.T, groupname string, version string, jobs ...clusterTe
 		}
 
 		t.Logf("determining old and new versions for upgrade testing addon %s", newAddon.GetName())
-		oldVersion, err := semver.Parse(strings.TrimPrefix(oldAddon.GetAnnotations()[constants.AddonRevisionAnnotation], "v"))
```
Comment (alejandroEsc) [inline test/addons_test.go:284]: Do we have any protection or failure in the case where we do not have this sent to use, i.e., revision is not included?

**Review by alejandroEsc — APPROVED** [review]
LGTM, one question but non-blocking.

File: test/addons_test.go (hunk 1 above)
Comment (shaneutt) [reply test/addons_test.go:284]: Yes, the tests will fail because the `semver.Parse()` will fail.

Comment (alejandroEsc) [reply test/addons_test.go:284]: Could we upgrade the revision annotation to a mandatory field soon?

Comment (shaneutt) [reply test/addons_test.go:284]: We should.

**Pull request merged**